buildkite/resource_pipeline.go

```diff
@@ -491,6 +499,15 @@ func updatePipelineExtraInfo(d *schema.ResourceData, client *Client) (PipelineEx
 	return pipelineExtraInfo, nil
 }
 
+func getTagsFromSchema(d *schema.ResourceData) []graphql.String {
+	tags := []graphql.String{}
+	eventSet := d.Get("events").(*schema.Set)
+	for _, v := range eventSet.List() {
+		tags = append(tags, graphql.String(v.(string)))
```

Comment:
`"events"` seems to be a typo as the schema has no mention of events. I think it should be "tags".

```suggestion
	tagSet := d.Get("tags").(*schema.Set)
	for _, v := range tagSet.List() {
		tags = append(tags, graphql.String(v.(string)))
```

Reply:
The type of the `tags` mutation input isn't a list of strings as well, it's a list of key-values.  
I discovered this from running the integration tests for this PR. Since we don't have tests set up for PRs at the moment, its a little hard to discover this yourself. So we can take it from here and get it merged in.

Please make sure you have given us edit permission on this PR though 👍

Reply:
You should have those permissions